databricks/koalas/tests/test_stats.py

```diff
@@ -122,3 +122,53 @@ def test_stats_on_boolean_series(self):
 
         self.assertAlmostEqual(ds.var(), s.var())
         self.assertAlmostEqual(ds.std(), s.std())
+
+    def test_some_stats_functions_should_discard_non_numeric_columns_by_default(self):
+        df = pd.DataFrame({'i': [0, 1, 2],
```

Comment:
The convention used for most tests is to use `pdf` for pandas DataFrames and `kdf` for Koalas DataFrames. It would be great if you could adapt those to make the code easier to read for other people who'll find `kdf` easier to understand than `ddf`.

The same applies to lines 147-150 and 162-165.

Reply:
Ok, I'll change that

Reply:
Although in this particular test file the convention is quite different. Do you want me to change them all to improve consistency across all tests ? Or should I change only the tests this commit adds ?

Reply:
You've already changed all variable names, right? 😉 But that's fine with me.

Reply:
Yes I did, but I figured I could always go back if you didn't like it :-)